modules/core/arrow-core-data/src/main/kotlin/arrow/core/Eval.kt

```diff
@@ -5,6 +5,8 @@ import arrow.higherkind
 fun <A> EvalOf<A>.value(): A = this.fix().value()
 
 /**
+ * ank_macro_hierarchy(arrow.core.Eval)
```

Comment:
Eval is not a type class so it does not need to render a hierarchy diagram

Reply:
Removed